### DISCUSSION TITLE
allurectl: Attachments not uploaded to Allure TestOps

### DISCUSSION THREAD

**Discussion by gvalamatsas-lnw** [allure-framework/allure2#2452] [TestOps]
Hello, we face an issue when we try to upload the allure results to Allure TestOps.
Specifically, the image and txt attachments are recognized as Orphan Files and so they aren't uploaded.
In the local Allure Report we can see the attachments when we do the allure serve:
![Screenshot 2024-03-27 at 12 47 15 PM](https://github.com/allure-framework/allure2/assets/152872696/b08b0989-2946-41cf-a609-dea1651712f1)

But when we upload the results to the TestOps, the images are not rendered:
![Screenshot 2024-03-27 at 12 48 21 PM](https://github.com/allure-framework/allure2/assets/152872696/7025dd08-d458-42e6-ae52-425863cb0380)

The command we run to upload the results is this one:
`allurectl upload --log trace --launch-name "${USER} Local PC $(date +"%Y-%m-%d %H:%M:%S")" AG.QA.Phoenix.PW.Test.Games/bin/Debug/net6.0/allure-results`
(We have also set the environment variables for the endpoint, project id, and API token)

Please find attached the allure-results folder, and the trace logs of the above command.
[allure-results.zip](https://github.com/allure-framework/allure2/files/14772155/allure-results.zip)
[allurectl-logs.log](https://github.com/allure-framework/allure2/files/14772177/allurectl-logs.log)

**Comment by cheshi-mantu** [comment]
Hi Giorgos,
for the Allure Testops ecosystem we have dedicated help desk located at https://help.qameta.io, please use this channel to get heko with Allure Testops and all stuff around it.
When registering there, please use the email address of your company, not a public email address.

**Reply by gvalamatsas-lnw** [reply]
Thank you cheshi, I've registered in the help desk and created the bug there
